src/Crdt.Sample/Changes/AddAntonymReferenceChange.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Crdt.Changes;
+using Crdt.Core;
+using Crdt.Entities;
+using Crdt.Sample.Models;
+
+namespace Crdt.Sample.Changes;
+
+public class AddAntonymReferenceChange(Guid entityId, Guid antonymId)
+    : EditChange<Word>(entityId), ISelfNamedType<AddAntonymReferenceChange>
+{
+    public Guid AntonymId { get; set; } = antonymId;
+
+    public override async ValueTask ApplyChange(Word entity, ChangeContext context)
+    {
+        if (!await context.IsObjectDeleted(AntonymId))
```

Comment:
I think this is a good place to add an in context comment about how we are handling the referential integrity issue.

Reply:
done, check the comment now